src/Uno.UI/Controls/CommandBarHelper.iOS.cs

```diff
@@ -33,7 +32,11 @@ public static void PageCreated(UIViewController pageController)
 				// For example the Uno.UI.Toolkit.CommandBarExtensions.BackButtonTitle attached property is often set globally to "" through
 				// a default CommandBar style in order to remove the back button text throughout an entire application.
 				// In order to leverage this information, we create a new CommandBar instance that only exists to "render" the NavigationItem.
-				topCommandBar = new CommandBar();
+				// Since Uno 3.0 objects which are not part of the Visualtree does not get the Global Styles applied. Hence the fact we are manually applying it here.
+				topCommandBar = new CommandBar
+				{
+					Style = Application.Current.Resources[typeof(CommandBar)] as Style
```

Comment:
Can we get the visual tree parent some other way ? (so that implicit styling can work and we can resolve the implicit style through the closest attached parent and not the app ?

/cc @davidjohnoliver

Reply:
We may be able to, but I still think it's not absolutely necessary. The app developer is not creating any CommandBar themselves, so why they would expect a locally-defined CommandBar style to be applied (to a page with no CommandBar, from the app's perspective)?

Reply:
I was wondering from a testing perspective, to avoid changing the CommandBar style for the whole sample application, and set it just in the tested sample.

Reply:
Right - the hope (not sure if @ajpinedam has already confirmed) is that the app-level style would only need to set an attached property, which would just be ignored by the UWP-styled CommandBar.

Reply:
And it would actually be possible to set the app-level style for the duration of the test only, ie by having a button that injected the style into Application.Current.Resources, and then removed it again when the test was completed.

Reply:
@davidjohnoliver  that is a wonderful idea. I just updated the UITest following your suggestion. We are now adding the GLobal Style for the CommandBar right before the UItest starts and remove it before leaving the UITest. 

By doing this we have encapsulated the Style to the execution of this UITest and preventing future issues with other UITests that could have applied the Style.